src/Microsoft.Diagnostics.Monitoring.RestServer/Auth/AuthConstants.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Microsoft.Diagnostics.Monitoring.RestServer
+{
+    internal static class AuthConstants
+    {
+        public const string PolicyName = "AuthorizedUserPolicy";
+        public const string NegotiateSchema = "Negotiate";
+        public const string NtlmSchema = "NTLM";
+        public const string ApiKeySchema = "HashedKey";
```

Comment:
I don't really like this name, but ApiKey seems very generic and likely used by other implementations.

Reply:
Renamed to MonitorApiKey